entry.go

```diff
@@ -163,18 +168,21 @@ func getCaller() *runtime.Frame {
 
 	// cache this package's fully-qualified name
 	callerInitOnce.Do(func() {
-		logrusPackage = getPackageName(runtime.FuncForPC(pcs[0]).Name())
-
 		// now that we have the cache, we can skip a minimum count of known-logrus functions
 		// XXX this is dubious, the number of frames may vary store an entry in a logger interface
-		minimumCallerDepth = knownLogrusFrames
+		if len(skipPackageNameForCaller) != 0 {
```

Comment:
Can you explain what is going on here? 
I have the impression `minimumCallerDepth = knownLogrusFrames` should remain the same.

Reply:
Hi @lavoiesl @tossp ,
won't this be unused since we are already including the logrus package in the ignore map?